project.clj

```diff
@@ -100,6 +100,18 @@
          "puppetserver"
          "vendor"]))
 
+(def pdb-aot-namespaces
+  (apply vector
+         #"puppetlabs.puppetdb\..*"
```

Comment:
Only one of the `.` is escaped here

Reply:
Yeah, that's intentional, i.e. we're looking for all the namespaces that start with "puppetlabs.puppetdb.".

Reply:
Hah, oh, uh, right.  Thanks.

Reply:
Looks like the remove below it might be wrong too.